test/b2gedmntuples_cfg.py

```diff
@@ -100,6 +102,7 @@
 process.load('Configuration.StandardSequences.MagneticField_38T_cff')
 process.load('Configuration.StandardSequences.Services_cff')
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
+process.load("RecoEgamma/PhotonIdentification/PhotonIDValueMapProducer_cfi")
```

Comment:
Hello Sal,

There is no need to add anything extra in the Read me prescription.
Everything works with the standard prescription.

Cheers,
Savvas

On Fri, Jul 10, 2015 at 1:56 AM, rappoccio <notifications@github.com> wrote:

> In test/b2gedmntuples_cfg.py
> <https://github.com/cmsb2g/B2GAnaFW/pull/11#discussion_r34317248>:
>
> > @@ -100,6 +102,7 @@
> >  process.load('Configuration.StandardSequences.MagneticField_38T_cff')
> >  process.load('Configuration.StandardSequences.Services_cff')
> >  process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
> > +process.load("RecoEgamma/PhotonIdentification/PhotonIDValueMapProducer_cfi")
>
> Hi, Savvas,
>
> For 3 I meant did you update the Readme with the prescription needed, or
> is this up to date?
>
> Cheers,
> Sal
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cmsb2g/B2GAnaFW/pull/11/files#r34317248>.
>